packages/go/cypher/analyzer/analyzer.go

```diff
@@ -72,29 +90,35 @@ func (s *ComplexityMeasure) onFunctionInvocation(node *model.FunctionInvocation)
 		// Calling for a relationship's type is highly likely to be inefficient and should add weight
 		s.Weight += Weight2
 	}
+
+	return nil
 }
 
-func (s *ComplexityMeasure) onQuantifier(node *model.Quantifier) {
+func (s *ComplexityMeasure) onQuantifier(stack *model.WalkStack, node *model.Quantifier) error {
 	// Quantifier expressions may increase the size of an inline projection to apply its contained filter and should
 	// be weighted
 	s.Weight += Weight1
+	return nil
 }
 
-func (s *ComplexityMeasure) onFilterExpression(node *model.FilterExpression) {
+func (s *ComplexityMeasure) onFilterExpression(stack *model.WalkStack, node *model.FilterExpression) error {
 	// Filter expressions convert directly into a filter in the query plan which may or may not take advantage
 	// of indexes and should be weighted accordingly
 	s.Weight += Weight1
+	return nil
 }
 
-func (s *ComplexityMeasure) onKindMatcher(node *model.KindMatcher) {
+func (s *ComplexityMeasure) onKindMatcher(stack *model.WalkStack, node *model.KindMatcher) error {
 	switch typedReference := node.Reference.(type) {
 	case *model.Variable:
 		// This kind matcher narrows a node reference's kind and will result in an indexed lookup
 		s.nodeLookupKinds[typedReference.Symbol] = s.nodeLookupKinds[typedReference.Symbol].Add(node.Kinds...)
 	}
+
+	return nil
 }
 
-func (s *ComplexityMeasure) onPatternPart(node *model.PatternPart) {
+func (s *ComplexityMeasure) onPatternPart(stack *model.WalkStack, node *model.PatternPart) error {
```

Comment:
I can't remember if it's a compiler warning or not but you may want to consider renaming "stack" to "_" since that param is unused.

Reply:
Patched up